test/unit/deletion_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'test_helper'
+
+class DeletionTest < ActiveSupport::TestCase
+  self.use_transactional_fixtures = false # Disabled to test after_commit
```

Comment:
We wont need to do this anymore on Rails 5.
I wonder if we should try to point to master, so rubygems.org could help the Rails team testing Rails 5 as it develops.